lab-brianna/test/server-test.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const server = require('../server');
+const chai = require('chai');
+const http = require('chai-http');
+const expect = chai.expect;
+
```

Comment:
**TODO FOR LOST POINTS**

You should have ``chai.use(http);`` here at line 7 before all your tests or else your ``chai.request(server)`` doesn't work at all.